docs/modules/reference/pages/glossary.adoc

```diff
@@ -37,20 +41,20 @@ Daemon:: A {page-component-title} sub-service that performs a specific set of ta
 Direct-discovered nodes:: Nodes that are added to {page-component-title} manually through a requisition.
 (See xref:operation:provisioning/directed-discovery.adoc[Directed Discovery].)
 
-Discourse:: Discourse is an open-source forum platform that hosts OpenNMS's community forums (https://opennms.discourse.group/).
+Discourse:: An open-source forum platform that hosts https://opennms.discourse.group/[OpenNMS's community forums].
 There, users can find knowledge base articles, support, announcements, and general discussion about all things OpenNMS.
 
 Discovery:: A process that uses ICMP to find network devices and services in your network and then automatically adds them to monitoring.
 (See xref:operation:provisioning/introduction.adoc#discovery-auto[Provisioning].)
```

Comment:
> Thoughts on this format of making the term a link instead of having the `(See ...)` at the end?

Hmm. That's a possibility. Not all the terms in the glossary will have links, so it might look odd. Also, there are some `(See ...)` links that aren't to the actual glossary term, but something related to it. 

The purpose of the link is to provide additional information beyond the defined term. Should we replace the `(See ..)` links that are for the same term with "Deep dive" or "More info" and a link, and keep the ones with related topics as `(See also ...)?

Reply:
I think the current :"See..." terminology for those items are fine.  Was just thinking that if the text of the reference matches the term or a phrase in the definition, it would be more concise to make the term/phrase a link. At least for xrefs to internal pages.  Anything that's a link outside the sites we control should stay as is.

Reply:
And making only some terms links seems to be acceptable within best practices for online glossaries.